apiserver/application/application.go

```diff
@@ -1219,9 +1239,6 @@ func (api *API) charmIcon(backend Backend, curl *charm.URL) ([]byte, error) {
 // relations.
 func (api *API) Consume(args params.ConsumeApplicationArgs) (params.ConsumeApplicationResults, error) {
 	var consumeResults params.ConsumeApplicationResults
-	if err := api.checkCanWrite(); err != nil {
```

Comment:
Again, this seems the wrong way around if this is in the consuming model - we still need write access there, right?